### ISSUE TITLE
Building the application inside OpenShift fails

### ISSUE THREAD

**Issue by adelton** [rh-aiservices-bu/metrobus-repairs-nlp-workshop#19]
Part 9 of https://developers.redhat.com/developer-sandbox/activities/use-rhods-to-master-nlp describes how to use this repository to build it into a Web application.

However, creating a Deployment fails, on current OCP 4.14.1 sandbox cluster.

After entering the git repository URL, a message

> Builder Image detected.
> A Builder Image is recommended.
> Python 3.9 (UBI 8)
> builder python

shows up.

When the build is then run, it fails with
```
ERROR: Could not find a version that satisfies the require....25.1, 1.25.2, 1.26.0b1, 1.26.0rc1, 1.26.0, 1.26.1, 1.26.2)
ERROR: No matching distribution found for numpy==1.19.2
WARNING: You are using pip version 21.3.1; however, version 23.3.1 is available.
You should consider upgrading via the '/opt/app-root/bin/python3.9 -m pip install --upgrade pip' command.
error: build error: building at STEP "RUN /usr/libexec/s2i/assemble": while running runtime: exit status 1
```

**Comment by frenzymadness** [comment]
I'm reposting the comment from @befeleme from the issue in s2i-python-container repository:

> So it seems the app was built either for Python 3.7 or Python 3.8 which is a bit unfortunate, as they're no longer supported.
> I managed to build and run the app on ubi8:python39 with those tweaks:
> 
> git diff requirements.txt:
> 
> -numpy==1.19.2
> +numpy==1.20.3
> +werkzeug>=2.2,<3.0
> 
> git diff nb-requirements.txt:
> 
> -numpy==1.19.2
> +numpy==1.20.3
> 
> The simple GET and POST requests to the app worked for me. However I can't tell whether upgrading numpy like that will work for every usage in your app.

**Comment by guimou** [comment]
Fixed by #20
